Continuous Delivery and Operation Excellence/Continuous Delivery.md

```diff
@@ -0,0 +1,65 @@
+# Content gathered for Continuous Delivery
+
+## Delivery formats
+
+* Webinar 
+* Self-service content 
+* Office Hours
+* Lab
+
+## Content Delivery Schedule
+
+Plan | Topic | Delivery Format 
+---- | ----- | ---------------
+Week 1 | Fast and Continuous Delivery in SaaS SDLC | Webinar
+Week 2 | Modernizing Compute | Webinar
+Week 3 | Application Storage Modernization | Webinar
+Week 4 | Innovation with Data & AI | Webinar
+Week 5 | Azure Marketplace | Webinar
+Week 6 | Identity in SaaS | Webinar
+Week 7 | The ISV Modernization Journey | Webinar
+Week 8 | Modern Architecture Patterns | Webinar
+
+
+## Plans Details
+
+### DevOps culture | Webinar | Kickoff for this item
+
+Goal of the Webinar:
+* DevOps Culture
+* Benefit of DevOps culture in related to SaaS Journey:
+  Ref the Continuous Delivery section in ISV SaaS Playbook(Grow Your ISV Business with SaaS: Microsoft Practices Development Playbook)
+* How DevOps and automation can improve productivity, security and fast go to market, A/B testing
+* Sharing resources
+* Microsoft tools & Open source tools sharing
+* Gather next interests
+* Asking for input on interests on Office Hours, frequency 
+* Asking for input of common challenges in speeding up SDLC
+
+## Top of minds & Reference resources
+
+Office Hours for Q&A on DevOps topics
+Office Hours for Q&A on Operation Excellence topics: Monitoring, Governance
+
+Challenges of streamline development processes of ISV:
+* A common platform for CI /CD
+* A pipeline, process which can be replicated and reuse across products and teams
+* Code qualities ensuring during development phrase
+* Infrastructure as Code habit and practices
+* Integration test and env for integration tests
+* TODO: add specific difficulties of ISV 

```

Comment:
Adding some specific difficulties of ISVs 

-  Creating a custom pipeline
-  Monitoring and Error logs with Pipeline
-  Different tools for those who are not using AzDevops and Github ( Jenkins/ Gitlab / Hashicorp , Vagran, Ansible etc )